patches/chromium/export_gin_v8platform_pageallocator_for_usage_outside_of_the_gin.patch

```diff
@@ -4,6 +4,10 @@ Date: Tue, 3 Nov 2020 16:49:32 -0800
 Subject: export gin::V8Platform::PageAllocator for usage outside of the gin
  platform
 
+In order for memory allocation in the main process node environment to be
+correctly tagged with MAP_JIT we need to use gins page allocator instead
+of the default V8 allocator.  This probably can't be usptreamed.
```

Comment:
Could we upstream something that converted the non-static method that already exists for this into a static method? Since it doesn't actually use any instance state.

Alternatively, can we call the instance method that already exists instead of adding this patch?

Reply:
@nornagon We can't call the instance method because we don't use the gin platform.  And constructing one has side effects we don't want.  Converting the member to be a static though seems something upstream might take reasonably 🤔